liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -351,9 +351,15 @@ protected void installFromFile() throws Exception {
             log.debug(MessageFormat.format(messages.getString("debug.request.refresh"), ""));
         }
 
-        if (refresh && installDirectory.exists()) {
+        if (refresh && installDirectory.exists() && installDirectory.isDirectory()) {
             log.info(MessageFormat.format(messages.getString("info.uninstalling.server.home"), installDirectory));
-            FileUtils.forceDelete(installDirectory);
+            // Delete everything in the install directory except usr directory
+            for(File f : installDirectory.listFiles()) {
+                if(f.isDirectory() && f.getCanonicalPath().equals(userDirectory.getCanonicalPath())) {}
```

Comment:
Since userDirectory is known outside the for loop, you could get the canonical path and store it in variable for comparison in the if statement.